.github/workflows/build.yml

```diff
@@ -4,13 +4,21 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
-
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      fail-fast: false
```

Comment:
This ensures other jobs are not cancelled if one fails ([see documentation](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast)).